schema.graphql

```diff
@@ -282,7 +282,6 @@ type Mutation {
 
 type Subscription {
   calculationCreated: Calculation!
-  calculationProgress: Calculation!
```

Comment:
question: should this be removed? Doesn't seem related to the other changes.

Reply:
Ehh, i have no idea where that comes from...